src/borg/testsuite/archiver/check_cmd.py

```diff
@@ -54,6 +54,41 @@ def test_check_usage(self):
         self.assert_not_in("archive1", output)
         self.assert_in("archive2", output)
 
+    def test_date_matching(self):
+        shutil.rmtree(self.repository_path)
```

Comment:
is that needed? usually the test setup/teardown cleans up the fs.

Reply:
I was trying to get rid of the first 2 archives that are created during test setup so that I can use the archives with specific timestamps only.

Reply:
ah, ok!